packages/@aws-cdk/aws-appmesh/lib/route-spec.ts

```diff
@@ -107,6 +178,51 @@ export interface GrpcRouteSpecOptions {
    * List of targets that traffic is routed to when a request matches the route
    */
   readonly weightedTargets: WeightedTarget[];
+
+  /**
+   * The retry policy
+   * @default - no retry policy
+   */
+  readonly retryPolicy?: GrpcRetryPolicy;
+}
+
+/** gRPC retry policy */
+export interface GrpcRetryPolicy extends HttpRetryPolicy {
+  /**
+   * gRPC events on which to retry
+   * @default - no retries for gRPC events
+   */
+  readonly grpcRetryEvents?: GrpcRetryEvent[];
+}
+
+/**
+ * gRPC events
+ */
+export enum GrpcRetryEvent {
+  /**
+   * Request was cancelled
+   */
+  CANCELLED = 'cancelled',
+
+  /**
+   * The deadline was exceeded
+   */
+  DEADLINE_EXCEEDED = 'deadline-exceeded',
+
+  /**
+   * Internal error
+   */
+  INTERNAL = 'internal',
+
+  /**
+   * Resource was exhausted
```

Comment:
What does this mean? I feel like more explanation is needed here.

Reply:
I think the error aligns with gRPC code 8/RESOURCE_EXHAUSTED, which per [gRPC documentation](https://grpc.github.io/grpc/core/md_doc_statuscodes.html) means "Some resource has been exhausted, perhaps a per-user quota, or perhaps the entire file system is out of space." I'm not sure if I'm allowed to copy-paste that verbatim from the docs I just referenced - if I'm not, maybe we can say, "A resource has been exhausted." Thoughts?

Reply:
How about adding that link to the gRPC documentation to the JSDocs of that field?

Reply:
Ok no problem. I added `@see` links to each of the enums.